data-processing/entities/definitions/types.py

```diff
@@ -103,7 +103,6 @@ class Definition(SerializableEntity):
     """
 
     text: str
-    sentence_id: str
```

Comment:
Do you have a particular reason that you removed the `sentence_id` field? I thought that it might be useful in the future to extract the relative position of the sentence in a paper, etc.

Reply:
Okay! I'll add it back in 👍  It wasn't currently being used in the pipeline, though I was too hasty in deleting it.